content/docs/reference/autocert.mdx

```diff
@@ -15,47 +17,42 @@ import TabItem from '@theme/TabItem';
 
 # Autocert Settings
 
-This reference covers all of Pomerium's **Autocert Settings**:
+Pomerium’s Autocert settings provide a certificate issuance and management service in accordance with the [automated certificate management environment](https://letsencrypt.org/2019/03/11/acme-protocol-ietf-standard.html) (ACME) protocol.
```

Comment:
Let's be a little careful with the phrasing here. To me this reads like it's saying Pomerium is the certificate issuer, but that isn't the case.

I'm also wondering about the ACME protocol link — that URL looks like an announcement about the ACME protocol but I'm not sure if it gives the most useful context on what it is. I'm not sure if there's a nice summary explanation of this somewhere (the [wikipedia page](https://en.wikipedia.org/wiki/Automatic_Certificate_Management_Environment) introduction isn't super helpful either). There's always the official RFC: https://datatracker.ietf.org/doc/html/rfc8555.

What do you think of this?

```suggestion
The Autocert settings configure Pomerium to manage public-facing TLS certificates automatically, by integrating with a certificate authority that supports the [ACME protocol](https://datatracker.ietf.org/doc/html/rfc8555), such as [Let's Encrypt](https://letsencrypt.org/).
```

Reply:
My reasoning originally was that the Let's Encrypt article links to the RFC, so the user would likely follow that path if they really wanted to  know. But, I didn't consider the fact that you can configure your preferred CA. Linking to the source of truth and ultimate authority here makes more sense.